File: go_backend/models/caja.go
package models

type Caja struct {
	ID           string  `json:"id"`
	EspacioID    string  `json:"espacio_id"`
	UsuarioID    string  `json:"usuario_id"`
	Nombre       string  `json:"nombre"`
	NombreNorm   string  `json:"-"`
	Descripcion  string  `json:"descripcion,omitempty"`
	CapacidadMax *int    `json:"capacidad_max,omitempty"`
	CreatedAt    Time    `json:"created_at"`
	UpdatedAt    Time    `json:"updated_at"`
}

File: flutter_app/lib/core/models/caja.dart
class Caja {
  final String id;
  final String espacioId;
  final String nombre;
  final String descripcion;
  final int? capacidadMax;

  Caja({
    required this.id,
    required this.espacioId,
    required this.nombre,
    this.descripcion = '',
    this.capacidadMax,
  });

  factory Caja.fromJson(Map<String, dynamic> json) => Caja(
        id: json['id'] as String,
        espacioId: json['espacio_id'] as String,
        nombre: json['nombre'] as String,
        descripcion: json['descripcion'] as String? ?? '',
        capacidadMax: json['capacidad_max'] as int?,
      );

  Map<String, dynamic> toJson() => {
        'id': id,
        'nombre': nombre,
      };
}
